apps/users/filters.py

```diff
@@ -52,7 +52,7 @@ def filter_long_time(self, queryset, name, value):
 
         if name == 'is_password_expired':
             key = 'date_password_last_updated'
-        elif name == 'long_time_no_login':
+        elif name == 'is_long_time_no_login':
             key = 'last_login'
         else:
             raise ValueError('Invalid filter name')
```

Comment:
The code checks for a missing "value" argument when filtering. It also doesn't handle edge cases like checking passwords expired and long time no login with different names appropriately in the future. For more robust logic, consider adding type hints and ensuring compatibility across all use-cases. Additionally, you could add error handling to gracefully manage inputs where there is unexpected data types or inappropriate filters applied (e.g., incorrect password expiration duration).